README.md

```diff
@@ -3,7 +3,6 @@
 A lightweight library built on top of Android Architecture Component ViewModel to simplify how UI Controllers (e.g., `Activity`, `Fragment` & `NavBackStackEntry`) retain instances on Android.
 
 - Eliminate `ViewModel` inheritance.
-- Eliminate `SavedStateHandle` complexity.
```

Comment:
I felt like this was already contemplated by the 4th point, but if you feel like this deserves the extra attention then it makes sense to keep it around.

Reply:
Hm, I like the emphasis of before (as it is a painful API to deal with). However, I'm good with the new one.

Reply:
Nah let's leave it as it was before, then!